### PR TITLE
Add support for scylla-enterprise packages

### PR DESCRIPTION
**Pull request by roydahan**

### CODE REVIEW

**Comment by roydahan** [comment]
@amoskong need to backport to all branches.

File: sdcm/tester.py [hunk 1]
```diff
@@ -1000,7 +1000,8 @@ def get_scylla_versions(self):
         try:
             versions_output = self.db_cluster.nodes[0].remoter.run('rpm -qa | grep scylla').stdout.splitlines()
             for line in versions_output:
-                for package in ['scylla-jmx', 'scylla-server', 'scylla-tools']:
```
Comment (amoskong) [inline sdcm/tester.py]: How about replacing the fixed package name with pattern?

```
+++ b/sdcm/tester.py
@@ -1003,7 +1003,7 @@ class ClusterTester(Test):
         try:
             versions_output = self.db_cluster.nodes[0].remoter.run('rpm -qa | grep scylla').stdout.splitlines()
             for line in versions_output:
-                for package in ['scylla-jmx', 'scylla-server', 'scylla-tools']:
+                for package in ['scylla[-\w]*-jmx', 'scylla[-\w]*-server', 'scylla[-\w]*-tools']:
                     match = re.search('(%s-(\S+)-(0.)?([0-9]{8,8}).(\w+).)' % package, line)
                     if match:
                         versions[package] = {'version': match.group(2),
```

Or get the right package prefix by scylla_pkg()?
```
+++ b/sdcm/tester.py
@@ -1003,7 +1003,8 @@ class ClusterTester(Test):
         try:
             versions_output = self.db_cluster.nodes[0].remoter.run('rpm -qa | grep scylla').stdout.splitlines()
             for line in versions_output:
-                for package in ['scylla-jmx', 'scylla-server', 'scylla-tools']:
+                for package in ['-jmx', '-server', '-tools']:
+                    package = self.db_cluster.nodes[0].scylla_pkg() + package
                     match = re.search('(%s-(\S+)-(0.)?([0-9]{8,8}).(\w+).)' % package, line)
                     if match:
                         versions[package] = {'version': match.group(2),
```

**Comment by amoskong** [comment]
This is only necessary for branch-enterprise, and I have a comment for this
PR.

**Comment by roydahan** [comment]
I tried a pattern that didn’t work well (for me).
The prefix may work, but need to be tested.
I needed that fix quick.

We can have it for enterprise branch only for now.

**Pull request merged**